docs/operating-scylla/admin-tools/select-from-mutation-fragments.rst

```diff
@@ -1,5 +1,5 @@
 ============================================
-SELECT * FROM MUTATION_FRAGMENTS() Statement
+Reading mutations fragments
 ============================================
```

Comment:
You possibly have to remove some of the `===`. I don't know if it matters or not, @annastuchlik?